src/GameWorldSimulator/NeoServer.Game.Common/Contracts/Services/IItemRequirementService.cs

```diff
@@ -13,9 +13,9 @@ public interface IItemRequirementService
 
 public readonly ref struct Requirement
 {
-    public int MinLevel { get; init; }
+    public uint MinLevel { get; init; }
```

Comment:
I think we can change the MinLevel e MinMagicLevel to ushort as max value is 65535

Reply:
removed on last commit: https://github.com/OpenCoreMMO/OpenCoreMMO/pull/665/commits/82314937fa9eeedb50263b395791ec30a500404b